src/ssl.c

```diff
@@ -53643,4 +53643,23 @@ int wolfSSL_set_ephemeral_key(WOLFSSL* ssl, int keyAlgo,
 
 #endif /* WOLFSSL_STATIC_EPHEMERAL */
 
+#if defined(OPENSSL_EXTRA)
+void wolfSSL_THREADID_current(WOLFSSL_CRYPTO_THREADID* id)
+{
+    (void)id;
+    return;
+}
+unsigned long wolfSSL_THREADID_hash(const WOLFSSL_CRYPTO_THREADID* id)
```

Comment:
Please add comment with function explanation and reason why wolfSSL only returns 0UL here.

Reply:
Added comment